### PR TITLE
Feature/select

### PR DESCRIPTION
**Pull request by john-angel**
CSS styles updated for Select component

### CODE REVIEW

File: packages/css/src/atoms/selects/select.scss [hunk 1]
```diff
@@ -18,7 +16,7 @@
     }
 
     &__selected {
-        width: 100%;
+        width: Min(100%, tokens.$width);
```
Comment (suabochica) [inline packages/css/src/atoms/selects/select.scss]: Please, follow the convention of CSS to favor lowercase, even in the `min()` function.

File: packages/css/src/atoms/selects/select.scss [hunk 2]
```diff
@@ -34,55 +32,50 @@
...
+
+            &__disabled {
+                color: tokens.$disabled-border-color;
+            }
         }
 
         &__arrow {
             display: inline-flex;
-            margin-left: auto;
+            margin-left: auto;
```
Comment (suabochica) [inline packages/css/src/atoms/selects/select.scss]: Remove additional spaces here :)

**Review by suabochica — COMMENTED** [review]
LGTM!

File: packages/css/src/atoms/selects/select.scss (hunk 1 above)
Comment (john-angel) [reply packages/css/src/atoms/selects/select.scss]: Resolved with https://github.com/bod-endava/e-toast-ds/pull/54/commits/b4c973acf364b3e2704dca279d0eb848e8ce8073

File: packages/css/src/atoms/selects/select.scss (hunk 2 above)
Comment (john-angel) [reply packages/css/src/atoms/selects/select.scss]: Resolved with https://github.com/bod-endava/e-toast-ds/pull/54/commits/b4c973acf364b3e2704dca279d0eb848e8ce8073

**Review by suabochica — APPROVED** [review]
LGTM!

**Pull request merged**